pkg/controller/core.oam.dev/v1alpha2/application/application_controller.go

```diff
@@ -282,7 +280,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		Reason:             condition.ReasonReconcileSuccess,
 	})
 	r.Recorder.Event(app, event.Normal(velatypes.ReasonDeployed, velatypes.MessageDeployed))
-	return r.gcResourceTrackers(logCtx, handler, phase, true, true)
+	return r.gcResourceTrackers(logCtx, handler, phase, true, false)
```

Comment:
why true -> false?

will it incompatible?

Reply:
Just change the variable from `isPatch` to `isUpdate`  to make it more easy to understand.